ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -24,8 +30,8 @@ time, mark, audio, video {
   padding: 0;
   border: 0;
   font-size: 100%;
-  font: inherit;
-  font-family: $kuiFontFamily;
+  font: inherit; /* 1 */
+  font-family: inherit; /* 1 */
```

Comment:
For two, where you have a long block of text I'd just separate it out.

```
.cssThing {
  color: white;
  background: blue;
  // Zoom and outline are set to reset Internet Explorer
  // and Firefox. Blah blah, long comment spans multiple lines.
  zoom: 1;
  outline: none;
}
```

For 90% of comments though, the explanation is pretty short and on one item. I can be swayed here if you want it uniform and above. I just find this works most of the time.

```
.cssThing {
  color: blue;
  background: red;
  zoom: 1; // Fix for IE 11.
}
```

The other benefit is this means that comments _above_ the selector mean they are actually commenting the selector itself.

```
// This is used when the popover does this weird thing.
.cssThing--modifier {
}
```